src/test/java/dk/sdu/se_f22/sortingmodule/range/rangepublic/GetterTests.java

```diff
@@ -0,0 +1,103 @@
+                    () -> assertThrows(InvalidFilterTypeException.class, doubleFilter::getUserMaxInstant),
+                    () -> assertThrows(InvalidFilterTypeException.class, doubleFilter::getUserMinInstant)
+            );
+        }
+    }
+
+    @Nested
+    @DisplayName("Getters on LongFilters")
+    class gettersOnLongFilters {
+        @Test
+        @DisplayName("get As Time")
+        void getAsTime() {
+            LongFilter longFilter = getTestFilter();
+
+            assertAll(
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getDbMaxInstant),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getDbMinInstant),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getUserMaxInstant),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getUserMinInstant)
+            );
+        }
+
+        @Test
+        @DisplayName("get as Double")
+        void getAsDouble() {
+            LongFilter longFilter = getTestFilter();
+
+            assertAll(
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getDbMaxDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getDbMinDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getUserMaxDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, longFilter::getUserMinDouble)
+            );
+        }
+    }
+
+
+    @Nested
+    @DisplayName("Getters on TimeFilters")
+    class gettersOnTimeFilters {
+
+        @Test
+        @DisplayName("Get as Long")
+        void getAsLong() {
+            TimeFilter timeFilter = TimeFilterTest.getTestFilter("publishedDate");
+
+            assertAll(
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getDbMaxLong),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getDbMinLong),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getUserMaxLong),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getUserMinLong)
+            );
+        }
+
+        @Test
+        @DisplayName("get as Double")
+        void getAsDouble() {
+            TimeFilter timeFilter = TimeFilterTest.getTestFilter("publishedDate");
+
+            assertAll(
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getDbMaxDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getDbMinDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getUserMaxDouble),
+                    () -> assertThrows(InvalidFilterTypeException.class, timeFilter::getUserMinDouble)
+            );
+        }
+    }
```

Comment:
Perhaps there should be an extra test for ensuring that the getters for correct values does not throw exceptions?

Reply:
Chime in with opinions. We excluded it because it is simple getters in the end

Reply:
think its fair to not include

Reply:
I disagree as these exceptions should be handled the getter passes it on.